test/e2e/apimachinery/aggregator.go

```diff
@@ -105,20 +91,6 @@ var _ = SIGDescribe("Aggregator", func() {
 
 })
 
-func cleanTest(client clientset.Interface, aggrclient *aggregatorclient.Clientset, namespace string) {
```

Comment:
do you still need to change this?

just asking to keep the diff small

Reply:
I thought I would take advantage of the automatic cleanup and only defer those resources that dont belong to the namespace. Also not doing it within the `AfterEach`, but I dont really have a strong opinion on it. If you think its best to keep the diff small I can bring the function back.

Reply:
if you only add a healthz probe, the change is clear and scoped, I can lgtm and I think that I can also approve
... with the other changes, I defer to people more familiar with the test, this test is really complex